versions/1/kibana/spec.yml

```diff
@@ -74,3 +74,12 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing ML module assets
+      type: folder
+      name: ml_module
```

Comment:
```suggestion
      name: ml
```

Reply:
I think the folder should be under kibana but named `ml_module`. There might be other ML assets in the future.

Reply:
Yes this is a mistake. The changes I had made to test https://github.com/elastic/kibana/pull/94950 use `ml_module`
Updated in https://github.com/elastic/package-spec/pull/152/commits/92fab4345350202c9c60613e4604d236bfc18144